.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: False
+
+language: python
+
+matrix:
+  include:
+    - python: 2.6
```

Comment:
Python 2.6 will likely fail since bloscpack depends on 'mock' and that doesn't support 2.6 at the moment. Suggest to drop support for it entirely.